### PR TITLE
Docs: document 'manager' and '_log' attrs of logging.Logging

### PR DESCRIPTION
**Pull request by erlend-aasland**
Authored by @AA-Turner 

----
:books: Documentation preview :books:: https://cpython-previews--108145.org.readthedocs.build/

### CODE REVIEW

**Review by AA-Turner — APPROVED** [review]
Feels cheating to approve something I wrote!

A

**Comment by erlend-aasland** [comment]
> Feels cheating to approve something I wrote!

Well, I approved it by making it a PR :)

**Review by vsajip — APPROVED** [review]

File: Doc/library/logging.rst [hunk 1]
```diff
@@ -1019,6 +1019,14 @@ information into logging calls. For a usage example, see the section on
       'extra'. The return value is a (*msg*, *kwargs*) tuple which has the
       (possibly modified) versions of the arguments passed in.
 
+   .. attribute:: manager
+
+      Delegates to the underlying :attr:`!manager`` on *logger*.
```
Comment (serhiy-storchaka) [inline Doc/library/logging.rst:1024]: Why the reference is silenced? Is the `manager` attribute documented? If not, then the proper fix is to document it.

Comment (erlend-aasland) [reply Doc/library/logging.rst:1024]: I created gh-108198 so we don't forget about it.

**Pull request merged**